### PR TITLE
[B] Metis: fix shortname (#8)

### LINKED ISSUE #7: Get ontologyShortName from ontology accession prefix
ontologyShortName should use the ontology accession prefix:

e.g. 
ontologyIRI: 'http://purl.obolibrary.org/obo/GO_0007568'
then
ontologyShortName: 'go'

Note: exceptions
* EDAM (need to check the prefix for: topic/operation/format). Return shortname = edam
* Orphanet (need to check the prefix for: Orphanet). Return shortname = ordo

### PR DESCRIPTION
**Pull request by DrWilliamsSteven**
This PR is to:
* add the correct ontologyShortName from the accession prefix

Also:
* add test for request error

Closes #7 
Relevant to repositive/agora#102

### CODE REVIEW

File: src/annotate.ts [hunk 1]
```diff
@@ -37,7 +37,8 @@ export default async function annotate(opts: { payload: { field: string, term: s
       return res;
     })
     .catch((err: any) => {
-      console.error('_request error: ' + err);
+      //console.error('_request error: ' + err);
+      throw new Error('_request error: ' + err);
```
Comment (DrWilliamsSteven) [inline src/annotate.ts]: @Istar-Eldritch Should this throw an error or just log it?

**Comment by DrWilliamsSteven** [comment]
CircleCI tests fail due to low coverage on branches for service.ts

I'll take a look and see if I can figure it out

**Comment by DrWilliamsSteven** [comment]
@Istar-Eldritch 
Tests fail due to low coverage on [these lines](https://github.com/repositive/metis/blob/c8a1fac2a999368e6727edd92ad2f98486b9284d/src/service.ts#L8-L10) of service.ts
How should you test this?

File: src/annotate.ts [hunk 2]
```diff
@@ -61,10 +66,9 @@ export default async function annotate(opts: { payload: { field: string, term: s
       ontologyShortName // shortname from IRI
 
     };
-
+    //console.log(result);
```
Comment (Istar-Eldritch) [inline src/annotate.ts]: I would avoid logging to the std output if possible during runtime.

Comment (DrWilliamsSteven) [reply src/annotate.ts]: Yes. it is commented out.

Comment (Istar-Eldritch) [reply src/annotate.ts]: Sorry 😅

**Comment by Istar-Eldritch** [comment]
Good to go 🐯

**Pull request merged**